src/test/java/com/nirima/jenkins/plugins/docker/DockerCloudTest.java

```diff
@@ -58,34 +58,14 @@ public void testConstructor_0_10_2() {
 
     private static final String LOG_MESSAGE = "Docker cloud requires a non-blank name after Jenkins 2.402";
 
-    @Issue("JENKINS-70729") // Warn if cloud name is empty
```

Comment:
@MarkEWaite I removed these two tests because the base class (`Cloud`) now throws an `IllegalArgumentException` that does not allow the creation of `DockerCloud`.

Reply:
Thanks @car-roll !  Has the base class been extended to handle upgrades on systems that already have a Docker cloud named with an empty string?  If not, that seems like it will be needed in order to not break upgrades.

Reply:
So the base class has not been extended to support empty string Cloud names, but support is still there as I did not touch the `readResolve` in `DockerCloud`. 

Just to clarify what I meant, I believe the code that these two tests are testing (`testConstructorWithEmptyName` and `testConstructorWithNullName`) are testing the constructor itself
https://github.com/jenkinsci/docker-plugin/blob/ccec39ca5e6f72ff98e7d1d33367dd104603df4c/src/main/java/com/nirima/jenkins/plugins/docker/DockerCloud.java#L136
However, with this new jenkins version, we are now failing on the `super(name)` call, so that log message will now never be called. Test-wise, I do not think I can reproduce this scenario with the new Jenkins version.

The only time now we will have to deal with a null name constructor should be in existing cloud names, which is tested in `testCopyConstructor`.